src/app/content/highlights/components/SummaryPopup/ChapterFilter.tsx

```diff
@@ -114,6 +116,7 @@ export default styled(ChapterFilter)`
   outline: none;
   max-height: 72rem;
   overflow: auto;
+  z-index: 2;
```

Comment:
this is being fixe on another card, make sure merge leaves us with only 1 z-index directive